pkg/scalers/external_scaler.go

```diff
@@ -3,11 +3,13 @@ package scalers
 import (
 	"context"
 	"fmt"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/go-logr/logr"
+	"github.com/kedacore/keda/v2/pkg/util"
```

Comment:
Could you move this import to the next block (together with `pb "github.com/kedacore/keda/v2/pkg/scalers/externalscaler"`)? The static checks are failing due to it

Reply:
Thanks @JorTurFer for poining that out :pray: